tests/install_testbed/create_resources.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash -ex
```

Comment:
This is OK with python modules, but these are independent bash scripts. All of them are invoked at this moment from the same script, but this may change, for example if the installation is split in two nodes or if they are called from different lines in cloud-init. Anyway, these scripts are not modules of the calling script but the calling script is a mere aggregation. I think that the header does not bother at all.